src/main/java/org/apache/maven/plugins/javadoc/AbstractJavadocMojo.java

```diff
@@ -6144,6 +6171,23 @@ private boolean containsWarnings( String output )
         }
     }
 
+    /**
+     * Determines whether the specified string is informational output of the Javadoc tool.
+     *
+     * @param str string to check
+     * @return true if informational output, false if not or cannot be determined
+     */
+    private boolean isInfoOutput( String str )
+    {
+        return str == null
+                || str.startsWith( "Loading source file " )
```

Comment:
I think this is a typo. See:
```
13392  [WARNING] Javadoc Warnings
13393  [WARNING] Loading source files for package com.mycompany.myapp...
13394 -[WARNING] Constructing Javadoc information...
13395 -[WARNING] Building index for all the packages and classes...
13396 -[WARNING] Standard Doclet version 17.0.3+7-1
13397 -[WARNING] Building tree for all the packages and classes...
13398 -[WARNING] Generating /var/osipovmi/Projekte/maven-javadoc-plugin/target/it/MJAVADOC-259/target/site/apidocs/com/mycompany/myapp/App.html...
13399 +[DEBUG] Constructing Javadoc information...
13400 +[DEBUG] Building index for all the packages and classes...
13401 +[DEBUG] Standard Doclet version 17.0.3+7-1
13402 +[DEBUG] Building tree for all the packages and classes...
13403 +[DEBUG] Generating /var/osipovmi/Projekte/maven-javadoc-plugin/target/it/MJAVADOC-259/target/site/apidocs/com/mycompany/myapp/App.html...
```

am I right?

Reply:
Thanks for your feedback @michael-o 

PR updated forced pushed.

Reply:
I would prefer the original prefixes from the resource bundles for consistency reasons.

Reply:
Ok. Valid point.

Reply:
Code amended. Hope I got it now :)